src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/config.properties

```diff
@@ -1,8 +1,8 @@
 bitbucket=Bitbucket
 bitbucket.admin.credentials=Project admin token
-bitbucket.server.name=Server name
-bitbucket.url=URL
+bitbucket.server.name=Instance name
+bitbucket.url=Instance URL
 bitbucket.credentials=Credentials
-bitbucket.test.connection=Test connection
+bitbucket.test.connection=Check credentials
```

Comment:
/nit IMO "Test connection" is better because we're also validating the instance URL, not just the credentials